_opensearch/install/docker.md

```diff
@@ -384,3 +384,112 @@ The environment variable `"DISABLE_SECURITY_DASHBOARDS_PLUGIN=true"` disables th
 You can't reverse this step as the security dashboards plugin is removed in the process.
 To re-enable security for OpenSearch Dashboards, start a new container and set `DISABLE_SECURITY_DASHBOARDS_PLUGIN` to false or leave it unset.
 {: .note}
+
+## Using Windows PowerShell to Install and Configure Window Subsystem for Linux (WSL)
+
+For Windows 10, Windows 11 and Windows Server 2019 (Version 1709) and later a development instance of Docker can be installed and configured on the Windows Subsystem for Linux (WSL) quickly using PowerShell. 
+
+To install Windows Subsystem for Linux run the following command in PowerShell as Admin and then restart:
+```
+WSL –-install
+```
+To install the Docker PowerShell module run following block of commands in PowerShell as Admin and restart when prompted:
+```
+Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Force
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+Install-Module -Name DockerMsftProvider -Repository PSGallery -Force
+```
+To download the Docker Deskop client for Windows run the following block of commands in PowerShell as Admin and restart when prompted:
```

Comment:
Does it make sense to link to the sample docker compose file on the same page and give the user two options: either create the docker-compose.yml file in the user's directory, or type it out in PowerShell using `r`n as newline characters? That way we don't duplicate the same docker-compose file on this page.